src/ui/public/courier/data_source/doc_source.js

```diff
@@ -1,3 +1,13 @@
+/**
+ * @name DocSource
+ *
+ * @description This class is tightly coupled with _doc_send_to_es. Its primary
```

Comment:
The relationship between this class and _doc_send_to_es doesn't really describe the class at all